CMakeLists.txt

```diff
@@ -242,10 +242,10 @@ install(
     COMPONENT "openvswitch"
     EXCLUDE_FROM_ALL
 )
-foreach (EXTRA_SCRIPT check_qemu_oom dewebsockify vnctest)
+foreach (EXTRA_SCRIPT script/check_qemu_oom script/dewebsockify script/vnctest)
     install(
         FILES "${EXTRA_SCRIPT}"
-        DESTINATION "${OS_AUTOINST_DATA_DIR}"
+        DESTINATION "${OS_AUTOINST_DATA_DIR}/script"
```

Comment:
You created a symlink for `check_qemu_oom`. This symlink should probably be installed here as well.

Reply:
I wanted the symlinks only for convenience for developers. Everything in OS_AUTOINST_DATA_DIR isn't supposed to be called by normal users and code that relies on `check_qemu_oom` calls it from the script sub directory. Hence I would not install the symlink, fine?